.github/tests/test_requirements.txt

```diff
@@ -0,0 +1,55 @@
+aiohttp==3.8.3
```

Comment:
Call it requirements.txt, more standard.

Ideally we only include those that we actually import directly.